app/initializers/model-fragments.js

```diff
@@ -1,4 +1,6 @@
-import _ from 'model-fragments';
+// Import the fragment base class to ensure monkey-patching side effects are
+// applied before any store instances are created :(
+import _ from 'model-fragments/fragment';
```

Comment:
You actually don't even need to do the `_ from` if you don't want to. `import 'model-fragments/fragment';` is valid. We actually do this once on the test-helper already. https://github.com/lytics/ember-data-model-fragments/blob/addonify/tests/test-helper.js#L1